### ISSUE TITLE
Compare tool shows diff when fields are identical

### ISSUE THREAD

**Issue by nicoweidner** [spdx/tools-java#68]
I executed
```
$ java -jar tools-java-1.1.0-jar-with-dependencies.jar CompareDocs compareOutput.xlsx ../testResources/SPDXJSONExample-v2.2.spdx.json ../testResources/SPDXJSONExample-v2.3.spdx.json
```

The resulting spreadsheet shows quite a few diffs even though the corresponding properties are identical. Not sure what the most efficient way of showing that is, so I'll attach two screenshots

![Screenshot from 2022-09-09 16-40-19](https://user-images.githubusercontent.com/66428403/189376483-a39a673e-9bbe-4fd3-8415-54666d522fcd.png)
![Screenshot from 2022-09-09 16-41-06](https://user-images.githubusercontent.com/66428403/189376491-0987033a-cc72-4c62-8e67-d2507ff14b6b.png)

**Comment by goneall** [comment]
@nicoweidner - Can you upload the 2 files you used to compare?  I'll take a look.

**Comment by nicoweidner** [comment]
I used these two files which are included in the repo - I should probably have made that more clear, sorry about that (the above command was executed in the `target` directory after building the tools, should have kept that part of the path):
- https://github.com/spdx/tools-java/blob/master/testResources/SPDXJSONExample-v2.2.spdx.json
- https://github.com/spdx/tools-java/blob/master/testResources/SPDXJSONExample-v2.3.spdx.json

**Comment by goneall** [comment]
Turns out there are differences in the files, but there are other issues masking what the issues are.

For the difference in licenses, there seems to be an encoding issue in SPDXJSONExample-v2.2.spdx.json file which is causing a missmatch of the LicenseRef2 which in turn causes a missmatch of any expression which uses that reference.

For the other differences, the two SPDX document have the same document namespace.  This is confusing the comparer and causing the wrong document information to be displayed.  You can work around this by manually changing the document namespace in either of the documents and re-running the compare.

PR #70 adds a check and the utility will now fail if there are any duplicate document namespaces.

I noticed some document level fields that were not different but were showing up as different - this has been resolved with PR #70.

**Comment by nicoweidner** [comment]
Thanks for looking into this! After modifying the namespace, the real (and different) values do show up in the packages list.
